helm-chart/renku-notebooks/values.yaml

```diff
@@ -99,7 +99,7 @@ serverOptions:
     range: [0, 1]
   defaultUrl:
     displayName: default landing URL of the notebook server
-    type: string
+    type: enum
```

Comment:
why? what if someone has a new thing like r shiny?

Reply:
We can define a "string" option (not implemented yet on the ui side), but then the options don't make sense. The original plan was to provide options that would translate into a dropdown.

Reply:
`enum` is correct, this was my mistake